jenkins/spark-nightly-build.sh

```diff
@@ -19,11 +19,11 @@ set -ex
 
 . jenkins/version-def.sh
 
-mvn -U -B -Pinclude-databricks clean deploy $MVN_URM_MIRROR -Dmaven.repo.local=$WORKSPACE/.m2
+mvn -U -B '-Pinclude-databricks,!snapshot-shims' clean deploy $MVN_URM_MIRROR -Dmaven.repo.local=$WORKSPACE/.m2
```

Comment:
this will deploy the nightly version without the snapshot shims to urm.  Which means devs and integration tests can't use it.  Our integration test builds rely on this now.